live-examples/css-examples/lists-and-counters/list-style.html

```diff
@@ -0,0 +1,58 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="list-style">
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">list-style: square;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">list-style: inside;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">list-style: url('/media/examples/rocket.svg');</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">list-style: none;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">list-style: georgian inside url('/media/examples/rocket.svg');</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">list-style: georgian outside url('/non-existent.svg');</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+</section>
+<div id="output" class="output large hidden">
```

Comment:
nit: we should have an empty line between `/section` and `div`.

Reply:
Hey @wbamberg, I added the empty line. 

In regards to the icon, I found it here:
https://publicdomainvectors.org/en/free-clipart/Cartoon-rocket-silhouette/56593.html

Let me know if that works for this project or if I need to look somewhere else.

Reply:
That's perfect!